index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Trial Project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <main id="main-doc">
+        <nav id="navbar">
+            <header>
+                <strong>Coding</strong>
```

Comment:
Leave the styling for the CSS, try to avoid HTML tags that give style. That way here it should have an h1 tag with a class = "header-title" for example.

Reply:
same here

Reply:
Again, your comment here is polite 👍, but the reason for the suggested change is quite not clear. "So you can create a class to do your required changes."?

How about something like **_"I would suggest here to do the styling in the CSS file, by doing this your styles could be shared across different HTML documents allowing reuseability."_**

Reply:
This is awesome :handshake: